test/fake_http_server.cc

```diff
@@ -42,15 +42,25 @@ FakeServer::~FakeServer() {
   server_thread_.join();
 }
 
+GetHandler FakeServer::Return(const std::string& body) {
```

Comment:
[Optional] I usually prepend `/*static*/` on the previous line, so:
```c++
/*static*/
GetHandler FakeServer::Return(const std::string& body) {
  ...
}
```

Reply:
Done.